tests/ui/pages/desktop/extensions.py

```diff
@@ -0,0 +1,60 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from pages.desktop.base import Base
+
+
+class Extensions(Base):
+
+    URL_TEMPLATE = 'extensions/'
+
+    _featured_addons_locator = (By.CLASS_NAME, 'FeaturedAddons')
+    _top_rated_locator = (By.CLASS_NAME, 'HighlyRatedAddons')
+    _title_locator = (By.CLASS_NAME, 'LandingPage-addonType-name')
+    _trending_addons_locator = (By.CLASS_NAME, 'TrendingAddons')
+
+    def wait_for_page_to_load(self):
+        self.wait.until(
+            lambda _: self.is_element_displayed(*self._title_locator))
+        el = self.find_element(*self._title_locator)
```

Comment:
pep08 doesn't like two-character variable names, let's just name this `element`?